Jenkinsfile

```diff
@@ -43,7 +43,7 @@ pipeline {
         }
         stage('Build OptaPlanner') {
             steps {
-                mavenCleanInstall('optaplanner', false, ['run-code-coverage'], '-Dfull')
+                mavenCleanInstall('optaplanner', false, ['run-code-coverage'], '-Dfull -Dmaven.test.failure.ignore=true')
```

Comment:
Note to self: remove once Kogito has Drools 7.49.
Until then, the tests will be failing and I would therefore not get to see Sonar.

Reply:
1) Can we add a "expiration condition" here? So basically a comment

    // TODO Remove when drools dependency is 7.49.0

Reply:
2) Or will this be removed before we merge this PR?

Reply:
It will.